.github/scripts/translation/make_translation.sh

```
@@ -0,0 +1,164 @@
+#!/bin/bash
+
+#
+# Check working directory
+#
+BASE_DIR=$(dirname $0)
+BASE_DIR=$( cd $BASE_DIR; pwd )
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)